package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "webpack -wd --config webpack.example.config.js",
     "prepublish": "NODE_ENV=production webpack -p",
-    "test": "karma start",
+    "test": "NODE_ENV=test karma start",
```

Comment:
Or maybe just write a permanent wrapper around CSSTransitionGroup and use that instead?